docs/Apple/release-notes.md

```diff
@@ -65,16 +65,15 @@ These release notes describe the new features and enhancements in each release o
 + (void)setPrivacyConfiguration:(nullable NSDictionary<NSString *, id> *)privacyConfig forType:(WS1TelemetryType)type;
 ```
 
-    - NOTES for this API -
+- NOTES for this API -
```

Comment:
Re-indented this section so markdown will not treat it as a code sample (which caused the long lines to need side-scrolling to read).